src/styles/global.tsx

```diff
@@ -0,0 +1,12 @@
+/** @jsxImportSource @emotion/react */
+import { Global, css } from '@emotion/react';
+
+const GlobalStyle = () => (
+  <Global
+    styles={css`
+      @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css');
```

Comment:
```suggestion
      @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css');
```

먼저, 외부 라이브러리는 위와 같이 버전을 지정해주는 게 안전합니다.
기존 코드는 별도 버전을 지정하지 않기 때문에 항상 '최신' 버전을 가져오게 되는데요.
이는 개발자가 의도하지 않은 변화를 일으키게 됩니다.
외부 리소스의 업데이트가 이뤄지게 되면, 분명 내 코드베이스는 예전과 동일한 코드임에도 이와 같은 의존성으로 인해 갑자기 UI가 깨지거나 작동하지 않을 수 있습니다. 버전을 고정하면 “오늘 빌드한 결과와, 내일 빌드한 결과가 동일”해져 안정성이 확보되고, 예측 가능하게 됩니다.
따라서 버전을 지정하도록 수정해주세요 :)

둘째, 외부 CDN 리소스를 사용하는게 안전한지에 대한 고민도 필요합니다.
jsdelivr 서버의 장애 발생이나 네트워크 차단 가능성 등을 고려하지 않을 수 없겠죠?
가용성이 중요한 서비스라면, 우리가 통제할 수 없는 의존성을 갖게 되는 건 위험할 수 있을 거에요.
코드 수정은 하지 않아도 괜찮습니다만, 실제 운영 환경이라면 어떻게 해야 할지 고민해보시는 것도 좋겠습니다.